scripts/crash-reports-since.py

```
@@ -0,0 +1,273 @@
+            missing = os.path.join(scratch, "never-written.txt")
+            if new_since_snapshot(missing, directory) == cmux_reports(directory):
+                print("  ok   a missing snapshot reports everything rather than nothing")
+            else:
+                print("  FAIL a missing snapshot did not fail loudly enough")
+                failures += 1
+
+    # The CLI must refuse a missing snapshot rather than report an empty, successful scan.
+    absent = os.path.join(tempfile.gettempdir(), "crash-reports-since-absent-snapshot.txt")
+    if os.path.exists(absent):
+        os.remove(absent)
+    probe = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)